src/screenSharing/serverViewScreenSharing.ts

```
@@ -69,19 +69,23 @@ export const setupServerViewDisplayMedia = (
-          try {
-            currentProvider.handleDisplayMediaRequest(cb);
-          } catch (error) {
-            console.error(
-              'Server view screen sharing: error in handler:',
-              error
-            );
-            cb({ video: false } as any);
-          }
-        },
-        { useSystemPicker: false }
+  (request, cb) => {
```

ESLint

[error] Insert `······`

(prettier/prettier)